docs/install/README.md

```diff
@@ -1028,8 +1028,8 @@ will run on the same Docker network which means that they will have routable IPs
 #### Create kind clusters
 Run `setup-kind-multicluster.sh` as follows:
 
-```sh
-./setup-kind-multicluster.sh --clusters 2 --kind-worker-nodes 4
+```console
+./scripts/setup-kind-multicluster.sh --clusters 2 --kind-worker-nodes 4
 ```
 
 When creating a cluster, kind generates a kubeconfig with the address of the API server 
```

Comment:
I t would be better to wait for cert-manager to be rolled out; after line 1056, add the following:

```console
kubectl rollout status deployment cert-manager-webhook -n cert-manager
```

Reply:
Good call. I've added this to each of the cert-manager install sections.